config/settings.py

```diff
@@ -46,6 +46,7 @@
     'django.contrib.staticfiles',
     'website.apps.WebsiteConfig',
     'cars.apps.CarsConfig',
+    'user',
```

Comment:
tu raczej powinno być 'user.apps.UserConfig', choć brakuje pliku apps.py, jest za to zbędny settings.py